nextflow_schema.json

```diff
@@ -152,7 +152,7 @@
                 },
                 "neg_ctrl_substrings": {
                     "type": "string",
```

Comment:
Could you add a `pattern` here to restrict the entries? Maybe just no whitespace? Or, maybe restrict to only alphanumeric, numbers, and a few characters like `_` and `-`? Not sure what the best restriction would be.